process-deploy/src/main/java/com/wipro/ats/bdre/pdeploy/ProcessDeployMain.java

```diff
@@ -24,9 +24,12 @@
  * main class to run ProcessDeploy
  */
 public class ProcessDeployMain {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1118)